HULKs/hulk
# HuLA DBus interface for efficiently querying battery status and IDs

## Introduced Changes

This PR adds a DBus interface to HuLA for efficiently querying the battery status and body/head IDs.

This reduces the load of the aliveness service on HuLA since it only interacts with HuLA when actual requests are received.

## ToDo / Known Issues

- [x] Replace session with system DBus (the session bus one will only work after the first user login and restarting aliveness)

## Ideas for Next Iterations (Not This PR)

None yet.

## How to Test

Build a new image using the branch of HULKs/meta-hulks#11 and flash it to a NAO (currently, the NAOs `27`-`32` are already flashed with this image).

Use `./pepsi aliveness` to check that everything works as expected.
